### DISCUSSION TITLE
Opening of a community discord server Supabase

### DISCUSSION THREAD

**Discussion by Olyno** [supabase/supabase#2316] [Show and tell]
Hi everyone!

~~I've been thinking about it for a while, and I didn't really know if it was a good idea for me to do it (since I'm not part of the Supabase team), but I figured "yolo" (You Only Live Once).~~

~~So today I'm announcing it publicly: I'm opening [a Supabase community discord server](https://discord.supabase.com/)! I'll be the only member at the moment, but I'll try to do my best to help you if you need it (or just talk about Supabase in general, that's cool too).~~

~~Of course, **this server is not official**. However, I would welcome various Supabase team members with open arms if they wish to join (and will get permissions if they wish to make it official).~~

~~To answer some questions you may 

> have:~~

~~**Why did I create a discord server?**~~

~~I thought it was a pity not to be able to discuss the project live, and to have to go through the "discussion" tab of github each time (even if I always got a quick answer, and I thank the community for that!) Also, I'd love to receive notifications of various Supabase updates (e.g. Supabase Studio development), which currently isn't really possible, unless YouTube gives the notifications, but that's very often very late.
I'd also like to be able to help potential new members to get to grips with Supabase if they wish, with different possible integrations.
And finally, I'd love to be able to do events around Supabase later on.~~

**Edit:** As @kiwicopple said, Supabase has now its own official Supabase discord server. I'm gonna close the one I created, and let you join the new one!

**You can join the server [by clicking here](https://discord.supabase.com/)**

**Comment by ChronSyn** [comment]
This is a great idea! There's already a subreddit (https://www.reddit.com/r/Supabase/), but more ways to chat with folks working with SB is welcome.

**Reply by Olyno** [reply]
I didn't know about the Reddit, good to know, thanks you for your share!

**Comment by alon710** [comment]


**Reply by Olyno** [reply]
Yes there is a community discord server, that's why I created this post 🙂

**Comment by kiwicopple** [comment]
Hey everyone - we now have an official (community-led) discord, with the help of @Olyno and a few other members of the community. 

👉  https://discord.supabase.com

**Reply by inno-elon** [reply]
invite expired now

**Reply by asithade** [reply]
Yeah this is expired

**Reply by devodii** [reply]
the current one works!
